GEOSldas_App/preprocess_ldas_routines.F90

```diff
@@ -2275,8 +2259,11 @@ subroutine optimize_latlon(fname_tilefile, N_proc_string, optimized_file, run_di
           endif
        enddo
        
-       if( k /=6 ) stop ("one or more processes may accross the face")
-      
+       if( k /=6 ) then
+          err_msg = 'one or more processes may across the face'
```

Comment:
@weiyuan-jiang : I don't understand the error message.  Please clarify

Reply:
> @weiyuan-jiang : I don't understand the error message. Please clarify
 If that condition (k/=6) is true, that means a processor ( more processors)  may have grid cells from different faces, which is not allowed.